english/coffee/coffee.md

```diff
@@ -31,6 +31,6 @@ After having your coffee do you want to:
 
 [Bake yourself something sweet to go with this delicious brew?](cookie/bake-cookies.md)
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.